.eslintignore

```diff
@@ -4,6 +4,7 @@ components/**/*.jsx
 !components/*/demo/*
 !.*.js
 # Docs templates
+*.md
```

Comment:
Why ignore md from eslint?

Reply:
Are we linting markdown file with eslint?
`components/table/index.en-US.md` are full of red lines..

If it's expected to lint markdown file with eslint, I will try to find some other workaround in another PR

Reply:
`npm run lint:demo`

Reply:
Cool, I will remove this ignore here.
And I do have some red lines in `components/table/demo/ajax.md` as well...
Will take a look when I have time.